3d_generation/maisi/scripts/infer_using_diff_model.py

```diff
@@ -0,0 +1,586 @@
+#!/usr/bin/env python
```

Comment:
I think all the source files need license headers?

Reply:
Hi @ericspod, I have updated the script. Please proceed. Thanks.

Reply:
I saw the license headers were added. Closing this thread for now.